lib/content-services/src/lib/node-comments/node-comments.module.ts

```diff
@@ -18,20 +18,26 @@
 import { NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
 import { NodeCommentsComponent } from './node-comments.component';
-import { ADF_COMMENTS_SERVICE, CoreModule } from '@alfresco/adf-core';
+import {
+    ADF_COMMENTS_SERVICE,
+    ADF_COMMENT_LIST_SERVICE,
+    CoreModule
+} from '@alfresco/adf-core';
 import { NodeCommentsService } from './services/node-comments.service';
+import { ContentCommentListService } from './services/content-comment-list.service';
 
 @NgModule({
-    imports: [
-        CommonModule,
-        CoreModule
-    ],
+    imports: [CommonModule, CoreModule],
     declarations: [NodeCommentsComponent],
     exports: [NodeCommentsComponent],
     providers: [
         {
             provide: ADF_COMMENTS_SERVICE,
             useClass: NodeCommentsService
+        },
+        {
+            provide: ADF_COMMENT_LIST_SERVICE,
```

Comment:
there's already a comments service above, maybe worth moving the user image api there? to avoid having a service with a single api

Reply:
I created a separate service because the component is injected solo in the process-services/process-comments component. Nevertheless, I can just merge those two services since the comment list service is really just one api call